LevelComponents/EntitySet.cpp

```diff
@@ -258,100 +66,4 @@ namespace LevelComponents
         return newtable;
     }
 
-    /// <summary>
-    /// Find an EntitySet which includes the entity with id.
-    /// </summary>
-    /// <param name="entityglobalId">
-    /// Entity global id.
-    /// </param>
-    EntitySetAndEntitylocalId EntitySet::EntitySetFromEntityID(int entityglobalId)
-    {
-        struct EntitySetAndEntitylocalId tmpEntitySetAndEntitylocalId;
-        if (entityglobalId < 0x11)
-        {
-            tmpEntitySetAndEntitylocalId.entitysetId = 1;
-            tmpEntitySetAndEntitylocalId.entitylocalId = -1;
-            return tmpEntitySetAndEntitylocalId;
-        }
-        for (int j = 1; j < 90; ++j)
-        {
-            int entitysetptr = ROMUtils::PointerFromData(WL4Constants::EntitySetInfoPointerTable + 4 * j);
-            int i = 0;
-            while (ROMUtils::CurrentFile[entitysetptr + 2 * i] != (unsigned char) 0)
-            {
-                unsigned char *entityidtmp = ROMUtils::CurrentFile + entitysetptr + 2 * i;
-                if (*entityidtmp == (unsigned char) entityglobalId)
-                {
-                    tmpEntitySetAndEntitylocalId.entitysetId = j;
-                    tmpEntitySetAndEntitylocalId.entitylocalId = i;
-                    return tmpEntitySetAndEntitylocalId;
-                }
-                i++;
-            }
-        }
-        memset(&tmpEntitySetAndEntitylocalId, 0, sizeof(tmpEntitySetAndEntitylocalId));
-        return tmpEntitySetAndEntitylocalId; // TODO: Error handling
-    }
-
-    /// <summary>
-    /// Get the first action set of a choosed entity by its global id.
-    /// </summary>
-    /// <param name="entityglobalId">
-    /// Entity global id.
-    /// </param>
-    int EntitySet::GetEntityFirstActionFrameSetPtr(int entityglobalId)
-    {
-        return EntitiesFirstActionFrameSetsPtrsData[entityglobalId];
-    }
-
-    /// <summary>
-    /// Get Entity Positional offset by its global id.
-    /// </summary>
-    /// <param name="entityglobalId">
-    /// Entity global id.
-    /// </param>
-    EntityPositionalOffset EntitySet::GetEntityPositionalOffset(int entityglobalId)
-    {
-        EntityPositionalOffset tmpEntityPositionalOffset;
-        tmpEntityPositionalOffset.XOffset = EntityPositinalOffset[2 * entityglobalId];
-        tmpEntityPositionalOffset.YOffset = EntityPositinalOffset[2 * entityglobalId + 1];
-        return tmpEntityPositionalOffset;
-    }
-
-    /// <summary>
-    /// Use one of the palette to render the whole Entityset for testing.
-    /// </summary>
-    /// <param name="paletteId">
-    /// palette id.
-    /// </param>
-    /// <returns>
-    /// the EntitySet rendered a pixmap.
-    /// </returns>
-    QPixmap EntitySet::GetPixmap(int paletteId)
```

Comment:
Just a comment to track that we must implement this rendering function as part of the PR. Please resolve this comment when you commit those changes.

Reply:
done.